code/modules/client/preferences.dm

```diff
@@ -1161,6 +1168,24 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						joblessrole = BERANDOMJOB
 					if(BERANDOMJOB)
 						joblessrole = RETURNTOLOBBY
+				SetChoices(user)//tegu edit alt job titles
+			if("alt_title")
+				var/job_title = href_list["job_title"]
+				var/titles_list = list(job_title)
+				var/datum/job/J = SSjob.GetJob(job_title)
+				if(user.client.prefs.exp[job_title] > (J.get_exp_req_amount() + 3000)) //If they have more than 50 hours (300 Minutes) past the required time needed for the job, give them access to the senior title
```

Comment:
Is timelocking job titles (which give IC benefit) good in any way? Heads of staff makes sense because there's IC benefit behind timelocked jobs, but it comes with a lot of extra responsibility, so the timelock is to make sure you can handle it. Here it's just "here you go, free stuff"

Reply:
From what I can tell, it was originally to differentiate between new-er players and ones that have been around a while